modules/apps/click-to-chat/click-to-chat-test/src/testFunctional/tests/ClickToChatInstanceSettings.testcase

```diff
@@ -252,36 +250,30 @@ definition {
 		AssertElementPresent(locator1 = "Providers#CHAT_CRISP");
 	}
 
-	@description = "This is a test for LPS-129049. Verify if it is possible to hide the chat provider"
+	@description = "This is a test for LPS-129042. Verify if it is possible to hide the chat provider"
 	@priority = 4
 	test HideChatProvider {
-		Select(
-			locator1 = "ClickToChatConfiguration#SITE_SETTINGS_STRATEGY",
-			value1 = "Always Override");
+		task ("When the user choose 'Always Override' option") {
+			Select(
+				locator1 = "ClickToChatConfiguration#SITE_SETTINGS_STRATEGY",
+				value1 = "Always Override");
+		}
 
-		AssertNotVisible(locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ID");
+		task ("Then chat provider ID and Account ID is hidden") {
+			AssertNotVisible(locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ID");
 
-		AssertNotVisible(locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ACCOUNT_ID");
+			AssertNotVisible(locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ACCOUNT_ID");
+		}
 	}
 
-	@description = "This is a test for LPS-129049. Verify Hubspot provider can be enabled and disabled."
+	@description = "This is a test for LPS-129042. Verify Hubspot provider can be enabled and disabled."
 	@priority = 5
 	test HubspotCanBeEnabledAndDisabled {
-		Select(
-			locator1 = "ClickToChatConfiguration#SITE_SETTINGS_STRATEGY",
-			value1 = "Always Inherit");
-
-		Select(
-			locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ID",
-			value1 = "Hubspot");
-
-		var clickToChatHubspotPassword = PropsUtil.get("click.to.chat.hubspot.password");
-
-		Type(
-			locator1 = "ClickToChatConfiguration#CHAT_PROVIDER_ACCOUNT_ID",
-			value1 = ${clickToChatHubspotPassword});
+		ClickToChatUtils.fillGoalsChat(
```

Comment:
Missing task blocks

Reply:
The test  that was missing tasks was: `HubspotCanBeEnabledAndDisabled` I updtaed this now.  And the test: `HubspotProviderKeepsEnabledAfterLogout` is has been updated in the other PR. Thank you Nikki.